README.md

```diff
@@ -34,12 +34,15 @@ dicomsort [options...] sourceDir targetDir/<patterns>
     [-f,--forceDelete] - remove source without confirmation
     [-k,--keepGoing] - report but ignore dupicate target files
     [-v,--verbose] - print diagnostics while processing
+    [-s,--symlink] - create a symlink to dicom files in sourceDir instead of copying them
```

Comment:
This should technically be a different commit since it's documentation for the other PR.

Reply:
Yes. I agree with that. I apologize. In the interest of full disclosure, there is a typo-bug fix in a print string as well.